src/olympia/tags/cron.py

```diff
@@ -1,31 +0,0 @@
-import cronjobs
-
-from olympia.addons.models import Addon
-from olympia.tags.models import AddonTag, Tag
-
-
-@cronjobs.register
-def tag_jetpacks():
-    # A temporary solution for singling out jetpacks on AMO.  See bug 580827
-    tags = (
-        ('jetpack', {
-            '_current_version__files__jetpack_version__isnull': False
-        }),
```

Comment:
Editors have various flags, including one for jetpack add-ons, but it's independent of the tag - I kept the jetpack flag for editors, and removed the restartless, replaced it with "requires restart" as per the frontend.